coral-collection/src/scripts/SelectableCollection.js

```diff
@@ -87,21 +87,26 @@ class SelectableCollection extends Collection {
    @protected
    */
   _getPreviousSelectable(item) {
-    let sibling = item.previousElementSibling;
+    const items = this.getAll();
+    let index = items.indexOf(item);
+    let sibling = index > 0 ? items[index - 1] : null;
```

Comment:
when the index = 0 , we return firstSelectableItem, is this expected?

Reply:
We return the item itself, if it is selectable, or the first selectable, I think this is expected.